src/main/java/com/meronat/iplog/storage/Storage.java

```diff
@@ -81,8 +81,8 @@ private void createTables() throws SQLException {
         try (Connection conn = getConnection()) {
 
             conn.prepareStatement("CREATE TABLE IF NOT EXISTS REGISTRY("
-                + " IP VARCHAR(45),"
-                + " ID VARCHAR(36),"
+                + " IP CHAR(45),"
```

Comment:
It's smart moving ID to `CHAR` but my understanding is that IP should be a `VARCHAR` still(if I am representing it as a string which I am) as IPv4 addresses are less than 45 characters and variable while I believe IPv6 addresses can be anywhere from 39 to 45 characters in length.